app/app.ts

```diff
@@ -1,29 +1,34 @@
-import * as Express from 'express';
```

Comment:
Good work on splitting the server & app. I hope you understand the value of it.

I just want to convey a small yet important thing -> `Your express server is not your application. Your application is much more than just a server.`

I hope people get it, one day 🤷‍♂